third_party/proton/proton/profile.py

```diff
@@ -3,13 +3,27 @@
 import os
 
 from triton._C.libproton import proton as libproton
+from triton.language import core as tl
+from triton.language.core import builtin
 from .hook import register_triton_hook, unregister_triton_hook
 from .flags import set_profiling_off, set_profiling_on, is_command_line
 from typing import Optional
+from . import language
+import warnings
 
 DEFAULT_PROFILE_NAME = "proton"
 
 
+class dev:
```

Comment:
We can take it out if we just invoke it with `proton.language.record`

Reply:
Updated